test/library/standard/CommDiagnostics/commDiagsTable.compopts

```diff
@@ -0,0 +1 @@
+-scommDiagsPrintUnstable=false
```

Comment:
I think you could just remove the compopts

Reply:
Oh, I didn't even think to look at what the default was, thanks (I guess I'm surprised that this is the default...).

Reply:
It's the default because AMOs are almost always unstable, and we felt it would be more friendly to have the default be not to show them.

Reply:
That still seems surprising to me in that I'd think a user would want to know whether comms happen or not by default and wouldn't care if the number fluctuates from run to run as long as they were getting a sense of the count and magnitude.  The stability question seems to mostly impact us from a testing perspective rather than an end-user (unless they were similarly trying to diff output against an expected value).

Reply:
The number fluctuates from run to run but more importantly, the atomic reads in the `waitFor(endCount)` spins absolutely swamp anything else, so the AMO counts are only useful when the counted region doesn't include anything that does such a `waitFor()` or, if it does, you're a runtime developer looking at bulk behavior.

Reply:
That's fair, but https://github.com/Cray/chapel-private/issues/1138 would help with that scenario.  I'll fork this off to a separate topic in any case.